recipe-client-addon/lib/Storage.jsm

```diff
@@ -29,6 +30,34 @@ function loadStorage() {
 }
 
 this.Storage = {
+  DURABILITY_NAMESPACE: '_storageDurability',
+  DURABILITY_KEY: 'durable',
+  async seedDurability(sandbox) {
+    const globalDurabilityStore = Storage.makeStorage(this.DURABILITY_NAMESPACE, sandbox);
+    let durability = await globalDurabilityStore.getItem(this.DURABILITY_KEY);
+
+    durability = parseInt(durability, 10);
```

Comment:
Oh, okay! That makes sense. We should be able to remove the parsing/isNaN-ing in that case.